File: app/src/main/java/dtu/projekt/phonefreedom/SettingsScreen.java
package dtu.projekt.phonefreedom;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.ArrayAdapter;
import android.widget.AutoCompleteTextView;

public class SettingsScreen extends AppCompatActivity {



    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_settings_screen);

        AutoCompleteTextView LangSet = (AutoCompleteTextView) findViewById(R.id.autoCompleteTextView);

        ArrayAdapter<String> myAdapter = new ArrayAdapter<String>(SettingsScreen.this,
                android.R.layout.simple_list_item_1, getResources().getStringArray(R.array.Animation_selector));
        myAdapter.setDropDownViewResource(R.layout.dropdown_item);
        LangSet.setAdapter(myAdapter);


    }
}

File: app/src/main/java/dtu/projekt/phonefreedom/SettingsScreen.kt
package dtu.projekt.phonefreedom

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import dtu.projekt.phonefreedom.R
import android.widget.AutoCompleteTextView
import android.widget.ArrayAdapter
import android.content.Intent
import android.content.ComponentName
import android.view.View
import android.widget.Button

class SettingsScreen : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_settings_screen)
        val LangSet = findViewById<View>(R.id.autoCompleteTextView) as AutoCompleteTextView
        val myAdapter = ArrayAdapter(
            this@SettingsScreen,
            android.R.layout.simple_list_item_1, resources.getStringArray(R.array.Languages)
        )
        myAdapter.setDropDownViewResource(R.layout.dropdown_item)
        LangSet.setAdapter(myAdapter)
        val DoNot = findViewById<View>(R.id.Do_not_disturb) as Button
        DoNot.setOnClickListener {
            val settingdnd = Intent()
            settingdnd.component = ComponentName(
                "com.android.settings",
                "com.android.settings.Settings\$ZenModeSettingsActivity"
            )
            startActivity(settingdnd)
        }
    }
}